assets/stylesheets/layout/_main.scss

```diff
@@ -583,3 +592,24 @@ body.newdash div.wordpress-com-extension-promo {
 html.no-scroll {
 	overflow: hidden;
 }
+
+.is-section-sites #primary,
```

Comment:
We shouldn't be using `is-x` classes by themselves, it should be `.layout.is-x`.